packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.scss

```diff
@@ -52,15 +57,14 @@ $groupHeader-ease-in-back: cubic-bezier(0.600, -0.280, 0.735, 0.045);
 
 .title {
   @include ms-font-xl;
-  padding: 14px 6px;
-  display: inline-block;
   cursor: pointer;
   outline: 0;
+  padding-left: 6px;
 }
 
 .expand {
   width: 36px;
-  height: 40px;
+  padding-top: 4px;
```

Comment:
Why is this necessary? Are you trying to center something? If so this is very wrong.